google/cloud/forseti/services/server.py

```diff
@@ -75,6 +80,15 @@ def serve(endpoint,
     if enable_console_log:
         logger.enable_console_log()
 
+    if enable_profiler:
+        try:
+            googlecloudprofiler.start(
+                service='forseti-server',
+                verbose=2,
```

Comment:
I think that it would be a good idea, to keep the placeholder for the `project_id`, with a comment that for dev workflow, it can be uncommented and used in the dev environment.
```
# project_id must be set if not running on GCP, such as your local dev environment.
# project_id='my-project-id',
```